src/view/actions/setConsent.jsx

```diff
@@ -310,8 +309,7 @@ const ConsentObject = ({ value, index }) => {
           <RadioGroupWithDataElement
             dataTestIdPrefix="gdprContainsPersonalData"
             name={`consent[${index}].gdprContainsPersonalData`}
-            label="GDPR Contains Personal Data"
-            description="Does the event data associated with this user contain personal data?"
```

Comment:
I changed FieldDescriptionAndError to derive its width based off its only child, rather than having to pass `width` to both `FieldDescriptionAndError` and the child of `FieldDescriptionAndError`.  I did this before you created RadioGroupWithDataElement. When I rebased on top of your changes, RadioGroupWithDataElement was feeding multiple children to FieldDescriptionAndError, so that was causing an error.

I debated on how to solve it, but I thought the UX was kind of cluttered with a description at the RadioGroupWithDataElement level:

![image](https://user-images.githubusercontent.com/210820/125351079-dcec5f80-e31c-11eb-8263-1b0e80ac2fed.png)

So I removed FieldDescriptionAndError from RadioGroupWithDataElement and just made the labels be what the descriptions were.

Reply:
👍  Thanks. I'll update the docs.

Reply:
@jonsnyder When you mentioned that this change requires a change to the documentation, I believe you were thinking that the labels were referenced in the documentation and those references needed be changed. Just so you're aware, I didn't see any label references from the documentation (I assume they would be [here](https://experienceleague.adobe.com/docs/experience-platform/edge/extension/action-types.html?lang=en#set-consent), so I don't think anything needs to be changed. If you're aware of where they may be referenced or if you were thinking about something different when you made your comment, let me know.